components/prism-solidity.js

```diff
@@ -0,0 +1,77 @@
+Prism.languages.solidity = Prism.languages.extend('clike', {
```

Comment:
On a really quick first look, Solidity appears to be closer to JavaScript than C-like. Don't you think you'd benefit from extending `javascript` directly?